components/Meeting.tsx

```diff
@@ -54,23 +63,27 @@ export default function Meeting(): JSX.Element {
   const participantsPerPage = Math.round(rows * columns);
 
   return (
-    <Flex
-      width="100%"
-      height="100%"
-      alignItems="center"
-      justifyContent="center"
-      direction={direction}
-    >
+    <Flex width="100%" height="100%" direction="row" position="relative">
       {spaceEndsAt && <Timer />}
-      <Center width={`${screenShareWidth}px`} maxHeight="100%">
-        <ScreenShareRenderer attach={attachScreenShare} />
-      </Center>
-      <Gallery
-        gap={gap}
-        width={galleryWidth}
-        height={galleryHeight}
-        participantsPerPage={participantsPerPage}
-      />
+      <Flex
+        maxWidth={availableWidth}
+        height="100%"
+        alignItems="center"
+        justifyContent="center"
+        direction={direction}
+        flexGrow={1}
+      >
+        <Center width={`${screenShareWidth}px`} maxHeight="100%">
+          <ScreenShareRenderer attach={attachScreenShare} />
+        </Center>
+        <Gallery
+          gap={gap}
+          width={galleryWidth}
+          height={galleryHeight}
+          participantsPerPage={participantsPerPage}
+        />
+      </Flex>
+      <ChatRenderer show={width > 800 && isChatOpen} />
```

Comment:
Interesting that the Chat panel just doesn't show up if the width is less than 800. Couldn't that be confusing if someone is on a smaller device?

Reply:
I was thinking that it would be difficult to see Chat as well as the participants' media in smaller screens, so I opted to hide chat in such cases. I'm open to any suggestions though.